### ISSUE TITLE
[🐞] V2 - “useTask$” only listens for the first change in the client.

### ISSUE THREAD

**Issue by devcaeg** [QwikDev/qwik#7134]
### Which component is affected?

Qwik Runtime

### Describe the bug

Using a “track” in a “useTask$” causes the client to hear only the first change in the “track” but not the other changes.

### Reproduction

https://stackblitz.com/edit/github-6f6maw-u2pxmf?file=src%2Froutes%2Findex.tsx

### Steps to reproduce

Make multiples clicks on the “Toggle errors” button, and you will see how only the first error is executed, and you will not hear anymore the change of errors.

### System Info

```shell
npmPackages:
    @qwik.dev/core: 2.0.0-alpha.1
    @qwik.dev/router: 2.0.0-alpha.1
    typescript: 5.7.2
    undici: 7.0.0 
    vite: 6.0.1
```

**Comment by JerryWu1234** [comment]
@Varixo are you fixing right now? May I pick this version 2 bug?

**Comment by JerryWu1234** [comment]
one Workaround is like this

```js
return {store, from};
```

@devcaeg 
I'm still working this bug,

**Comment by devcaeg** [comment]
Yes, that's the way it works. But anyway plugins like modular-forms uses `{ store, [ Form, ... ] }`

**Comment by Varixo** [comment]
https://qwikdev-build-v2.qwik-8nx.pages.dev/playground/#f=7VaxDoIwEN39ituAxJAwGCMRnJhNkI2wqauJsjX3717vChxpRR3cZGra8ui9Pu49JZpttgmJZj1aLo%2B4rb2PJ1K2%2BHCh0PaG2tPtnrs8gKVlIOYoMXIDky3HxpksqlWQOadLfjdlVCpJ1rXz%2B%2BtWqzzQ2yI5q0ggSmTD7Jb0PTleGRBXFu%2BT30e%2B2EPLkagjYtqB21h4IXkaxKTjAy6zoDlgPKnxkfZkdKEKQ5uEjKXAKGlvCjlGl2MfD5bK8ucOQO58uZJJnCGHqKrrYw1NdWpIM4KDWLJzjAFzOIFcDIMVxkPGQeM6vL7uRtlO5e1vutFM6P9e9Ite9AQ

**Comment by Varixo** [comment]
fixed by #7217
